### PR TITLE
upgrade: prepping upgrade to v1.2.0

### PR DESCRIPTION
**Pull request by jumpdest7d**
Please review my script changes @quasystaty1

### CODE REVIEW

**Review by quasystaty1 — COMMENTED** [review]
Looks great, Just add the node for the flag and you should be able to use it.
In line 11 set NODE="https://archive-rpc.noria.nextnet.zone:443"
Note: The script will also prompt you to vote on the proposal.

**Review by matpb — APPROVED** [review]
Don't forget to run shell scripts through https://www.shellcheck.net/, it's magic.
There is a small potential issue with $NODE which could potentially contain shell characters and I wonder if it wouldn't be better to to wrap its usage in double-quotes?

Anyway, nothing really problematic, I'm all in.

File: scripts/chain_upgrade.sh [hunk 1]
```diff
@@ -2,21 +2,29 @@
...
-NODE="TBA"
 export DAEMON_NAME="noriad"
 export DAEMON_HOME="$HOME/$BINARY_DIR"
 
-BINARY_DOWNLOAD_URL="https://github.com/noria-net/noria/releases/download/v0.x.x/noria_linux_amd64.tar.gz?checksum=sha256:xxxx"
+# Get NODE value from the input argument or use the default value
+if [ -n "$3" ]; then
+  NODE=$3
+else
+  NODE=$(grep -oP '(?<=node\s=\s")[^"]+' "$DAEMON_HOME/config/client.toml")
```
Comment (matpb) [inline scripts/chain_upgrade.sh:18]: When wrapping vars in double quotes, you'd usually want to wrap the var only:
`"$DAEMON_HOME"/config/client.toml`

Here is the docs behind: https://www.shellcheck.net/wiki/SC2086

PS: this should still work fine, I'm just sharing for best standards purposes

File: scripts/chain_upgrade.sh [hunk 2]
```diff
@@ -30,14 +38,14 @@ $DAEMON_NAME tx gov submit-proposal software-upgrade $UPGRADE_NAME \
   --from $KEY_NAME \
   --chain-id $CHAIN_ID \
   --home $DAEMON_HOME \
-  --node $NODE \
+  --node "$NODE" \
```
Comment (matpb) [inline scripts/chain_upgrade.sh:41]: Yes :)

**Pull request merged**